UnityPy/files/SerializedFile.py

```diff
@@ -368,10 +368,10 @@ def read_type_tree_blob(self, type_tree):
 
         return string_buffer_reader.bytes
 
-    def get_writeable_cab(self, name: str = "CAB-UnityPy_Mod.resS"):
+    def get_writeable_cab(self, name: str = None):
```

Comment:
Since ``get_writeable_cab`` can be called without assigning ``name``,
a suiting name should be found by the function itself.
The name shouldn't be one of the existing resource cabs that are supposed to be read-only.

Reply:
Yeah, that was from my personal fork, forgot to revert.